src/pages/billing/forms/UpdatePaymentForm.js

```
@@ -23,9 +24,13 @@ export class UpdatePaymentForm extends Component {
 
     const newValues = values.card ? { ...values, card: prepareCardInfo(values.card) } : values;
 
-    return billingUpdate(newValues).then(() => {
-      showAlert({ type: 'success', message: 'Payment Information Updated' });
-    });
+    return billingUpdate(newValues)
+      .then(() => {
+        showAlert({ type: 'success', message: 'Payment Information Updated' });
+      })
+      .catch(error => {
+        ErrorTracker.report('update-payment-information-error', error);
```

Do we need this?

---

May be not because we are already reporting this error. An empty catch block then?